### PR TITLE
Improve command information in the readme

### PR DESCRIPTION
**Pull request by jsancio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,6 +132,10 @@ element in the array is a line in the requirements file.
 ```
 _Sample `command.json`._
 
+See the [Command Schema](
+https://github.com/mesosphere/universe/blob/master/repo/meta/schema/command-schema.json) for a
+detail description of the schema.
```
Comment (ConnorDoyle) [inline README.md]: s/detail/detailed

File: README.md [hunk 2]
```diff
@@ -132,6 +132,10 @@ element in the array is a line in the requirements file.
 ```
 _Sample `command.json`._
 
+See the [Command Schema](
+https://github.com/mesosphere/universe/blob/master/repo/meta/schema/command-schema.json) for a
```
Comment (ConnorDoyle) [inline README.md]: Could we use the relative form of the link for this?  See https://github.com/blog/1395-relative-links-in-markup-files

**Comment by ConnorDoyle** [comment]
LGTM, mod comments. :guitar:

File: README.md (hunk 1 above)
Comment (jsancio) [inline README.md]: Done.

File: README.md (hunk 2 above)
Comment (jsancio) [inline README.md]: Cool feature! Done.

**Pull request merged**